build/build.xml

```diff
@@ -622,6 +622,47 @@
 		<generate-wiki addon="httpsinfo" />
 	</target>
 	
+	<macrodef name="build-imagelocationscanner-addon" description="Builds imagelocationscanner add-on">
+		<sequential>
+			<build-addon name="imagelocationscanner">
+				<!-- Add the extra classes required -->
+				<jar jarfile="${dist}/${file}" update="true" compress="true">
+					<zipfileset dir="${build}" prefix="">
+						<include name="com/veggiespam/imagelocationscanner/**" />
+					</zipfileset>
+				</jar>
+			</build-addon>
+		</sequential>
+	</macrodef>
+
+	<macrodef name="build-imagelocationscanner-addon-without-help-indexes" description="Builds imagelocationscanner add-on">
+		<sequential>
+			<build-addon-without-help-indexes name="imagelocationscanner">
+				<!-- Add the extra classes required -->
+				<jar jarfile="${dist}/${file}" update="true" compress="true">
+					<zipfileset dir="${build}" prefix="">
+						<include name="com/veggiespam/imagelocationscanner/**" />
+					</zipfileset>
+				</jar>
+			</build-addon-without-help-indexes>
+		</sequential>
+	</macrodef>
+	
+	<target name="deploy-imagelocationscanner" description="deploy the Image Location and Privacy scanner addon">
```

Comment:
The "depends" that's in `deploy-pscanrulesAlpha` needs to be moved here (and from `deploy-pscanrulesAlpha-without-help-indexes` to following target).

The `build-all` needs to be updated to call the new macrodef and replace `build-imagelocationscanner-addon`.